src/main/bridge.js

```diff
@@ -1,100 +1,89 @@
 // @flow
 
 import '@babel/polyfill'
+import invariant from 'invariant'
 import { fork } from 'child_process'
-import { BrowserWindow, ipcMain, app } from 'electron'
-import objectPath from 'object-path'
+import { ipcMain, app } from 'electron'
+import { ipcMainListenReceiveCommands } from 'helpers/ipc'
 import path from 'path'
 
 import setupAutoUpdater, { quitAndInstall } from './autoUpdate'
 
-const { DEV_TOOLS } = process.env
-
 // sqlite files will be located in the app local data folder
 const LEDGER_LIVE_SQLITE_PATH = path.resolve(app.getPath('userData'), 'sqlite')
 
-const processes = []
+let internalProcess
```

Comment:
so that means that we no longer are able to have multiple internal processes. how about keeping the ability to do it?

Reply:
there is one internal process to run everything. since everything is asynchronous it shouldn't be a hurdle. if there are CPU intensive work, then yeah we need to introduce the concept of having a target process.
like you want to run that command on "internal", or on "main" or on "whatever".

but the previous way was too crazy because it was forking a process for each single event message

Reply:
what's important tho is that the commands that need the device have to run inside the same process.
and i think all our commands today uses the device so we have to do it like this